docassemble/MAPetitionToSealEviction/data/questions/petition_to_seal_eviction.yml

```
@@ -253,27 +253,26 @@ subquestion: |
   If you are a tenant whose landlord filed an eviction case against you, 
   you can use this interview to ask a judge to seal the court information.
 
-  Not all tenants qualify to seal their eviction records. This interview will:
+  Not all tenants qualify to seal their eviction records. This interview should not be used by commercial tenants seeking to seal an eviction:
 
-  * Help you understand if you can seal your eviction records.
-  * Make a "Petition to Seal eviction" for you to file with the court.
+  * Help you understand if you can seal your eviction record using this interview.
+  * Fill-out a Petition to Seal Eviction Record Pursuant to G.L. c. 239, § 16 to file with the court.
 
   Before you get started, please gather:
     
-  1. The name of the court that your eviction action was heard in
-  1. Information about the reason for your eviction
-  1. The date of the **final decision** (judgment) in your eviction action
-  1. Information about any other eviction cases that you have had since the case you are trying to seal.
+  1. The <a href="https://www.mass.gov/info-details/how-to-search-court-dockets?_gl=1*1t4lg0v*_ga*Mjc1NTYwNDQxLjE3NDE2MjkwMjA.*_" style="text-decoration: none; color: green; font-weight: bold;">docket number</a> of the eviction case you want to seal
```

```suggestion
  1. The docket number of the eviction case you want to seal (<a href="https://www.mass.gov/info-details/how-to-search-court-dockets?_gl=1*1t4lg0v*_ga*Mjc1NTYwNDQxLjE3NDE2MjkwMjA.*_">find your docket number in a new tab</a>)
```

1. Need to warn about external links
2. Links should always use the same style. Since this opens in a new tab, green link style isn't appropriate.

(did the court mean to make this a reference term, like the "green word" on the front page?) https://docassemble.org/docs/initial.html#terms

If that's the case, we just need to add a definition of Docket Number, not an external link.

---

Agree, I just went with their styling preference but it did seem counter intuitive to make it green. However, their instructions were to make it a hyperlink.

---

Something like "it's a combination of letters and numbers uniquely identifying your case. It should be written on the paperwork you got from the court or your landlord"